.github/workflows/keyboard.yml

```
@@ -48,43 +33,9 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-nodekbd-
             nodekbd-
-      - name: Build with Maven
-        run: >
-          mvn -s .github/workflows/mvn-settings.xml -B compile install package --file tools/pom.xml
-          -DskipTests=true
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: BOM check in .xml files
-        run: >
-          mvn -s .github/workflows/mvn-settings.xml --file tools/pom.xml -pl cldr-code
-          exec:java -Dexec.mainClass=org.unicode.cldr.util.XMLValidator
-          -Dexec.cleanupDaemonThreads=false
-          -Dexec.args="-q --justCheckBom keyboards"
-          && echo 'No XML BOM detected'
-      - name: Keyboard 3.0 Check
-        run: >
-          mvn -s .github/workflows/mvn-settings.xml --file tools/pom.xml -pl cldr-code
-          exec:java -Dexec.mainClass=org.unicode.cldr.util.XMLValidator
-          -Dexec.cleanupDaemonThreads=false
-          -Dexec.args="--schema=keyboards/dtd/ldmlKeyboard.xsd keyboards/3.0"
-          && echo 'No XML BOM detected'
-      - name: Keyboard Test 3.0 Check
-        run: >
-          mvn -s .github/workflows/mvn-settings.xml --file tools/pom.xml -pl cldr-code
-          exec:java -Dexec.mainClass=org.unicode.cldr.util.XMLValidator
-          -Dexec.cleanupDaemonThreads=false
-          -Dexec.args="--schema=keyboards/dtd/ldmlKeyboardTest.xsd keyboards/test"
-          && echo 'No XML BOM detected'
-      - name: Upload cldr-code.jar
-        uses: actions/upload-artifact@v2
-        with:
-          name: cldr-code
-          path: tools/cldr-code/target/cldr-code.jar
-      - name: Test with maven
-        run: >
-          mvn -s .github/workflows/mvn-settings.xml -B test --file tools/pom.xml
-          -DCLDR_GITHUB_ANNOTATIONS=true
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install kmc
+        run: npm install -g @keymanapp/kmc@alpha
+      - name: Compile Keyboards
+        run: kmc build keyboards/3.0/*.xml
```

```suggestion
        run: kmc --error-reporting build keyboards/3.0/*.xml
```

Is it okay to try and capture fatal errors to Keyman's Sentry instance?

---

I'll be rebasing once data pr lands but accepting for now as a test